src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
This package is copied from the h2o project directly:

https://github.com/h2oai/h2o-3/tree/master/h2o-genmodel/src/main/java/hex/genmodel/easy

Incidentally, this is also the reason I had to remove the header check. 

I'm new to the world of java packaging & dependency management, so any insight on how to properly handle this is appreciated. I imagine I could create a separate repo that tracks that package in the h2o repo, to use as an external dependency here, but I'm not sure if there is a better solution?